demo/scenario/Auctioneer1.createAuction.json

```diff
@@ -78,7 +78,96 @@
 			"name": "f",
 			"impairment": 30
 		}]
-	}], 
+
+	}, {
+                "id": 3,
+                "name": "Mudberg",
```

Comment:
fine adding the territories for better visual look, yet... it looks complicated again with 10 channels each! How about 1 channel each?

Reply:
Which view are you referring by "looks complicated"?  I thought currently in demo we don't show the createAuction and only the final result would show channels, a view i haven't seen yet.   That said, the number of channels looks in general high but i think we might want to reduce it overall more uniformally down to something like 3-5 rather than having some 10 channel and some 1 channel territories?  In any case, I think we will have to check it with the actual views (and based on some expected screen real estate (e.g., what resolution do we expect with what font-size)?